source/Octopus.Tentacle.Tests/Capabilities/CapabilitiesServiceV2Fixture.cs

```diff
@@ -18,8 +20,27 @@ public async Task CapabilitiesAreReturned()
             capabilities.Should().Contain("IScriptService");
             capabilities.Should().Contain("IFileTransferService");
             capabilities.Should().Contain("IScriptServiceV2");
-            capabilities.Should().Contain("IScriptServiceV3Alpha");
-            capabilities.Count.Should().Be(4);
+            capabilities.Count.Should().Be(3);
```

Comment:
can all these assertions be rewritten as 

`capabilities.Should().BeEquivalentTo("IScriptService", "IFileTransferService", "IScriptServiceV2");`

Reply:
Yes they can. They can also use `nameof()` so I'll change to use that

Reply:
fixed in [8f59057](https://github.com/OctopusDeploy/OctopusTentacle/pull/859/commits/8f59057fe87be8cb8c032d04ec46aa801b3ef9b9)